tool/README.md

```diff
@@ -71,93 +88,121 @@ From the git GUI tool or from github.com directly:
-and use this commit hash for the following steps.
-
-Once the build is in good shape, you can revert any local changes and proceed to the next step.
-```shell
-git checkout .
-git clean -f -d
-```
+- Checkout the commit you just created,
+   - or remain on the branch you just landed the prep PR from.
+   ```shell
+   git checkout 8881a7caa9067471008a8e00750b161f53cdb843
+   ```
+
+- Build the DevTools binary and run it from your local Dart SDK.
+   - From the main devtools/ directory.
+   ```shell
+   dart ./tool/build_e2e.dart
+   ```
+
+- Launch DevTools and verify that everything generally works.
+   - open the page in a browser (http://localhost:53432)
+   - `flutter run` an application
+   - connect to the running app from DevTools
+   - verify:
+      - pages generally work
+      - there are no exceptions in the chrome devtools log
+   - If you find any release blocking issues:
+      - fix them before releasing.
+      - Then grab the latest commit hash that includes
+         - the release prep commit
+         - the bug fixes,
+      - use this commit hash for the following steps.
+
+- Once the build is in good shape,
+   - revert any local changes.
+      ```shell
+      git checkout . && \
+      git clean -f -d;
+      ```
 
 ### Tag the release
-Checkout the commit from which you want to release DevTools (likely the
-commit for the PR you just landed). You can run `git log -v` to see the commits.
-Run the `tag_version.sh` script to create a tag on the `flutter/devtools` repo for this
-release. This script will automatically determine the version from `packages/devtools/pubspec.yaml`
-so there is no need to manually enter the version.
-```shell
-tool/tag_version.sh
-```
+- Checkout the commit from which you want to release DevTools
+   - This is likely the commit for the PR you just landed
+   - You can run `git log -v` to see the commits.
+- Run the `tag_version.sh` script
+   - this creates a tag on the `flutter/devtools` repo for this release.
+   - This script will automatically determine the version from `packages/devtools/pubspec.yaml` so there is no need to manually enter the version.
 
-### Upload the DevTools binary to CIPD
-Using the commit hash you want to release DevTools from (this should match the
-commit hash for the tag you just created) and the [update.sh](https://github.com/dart-lang/sdk/blob/master/third_party/devtools/update.sh)
-script, build and upload the DevTools binary to CIPD.
+   ```shell
+   tool/tag_version.sh;
+   ```
 
-```shell
-cd $LOCAL_DART_SDK
-git rebase-update
-third_party/devtools/update.sh <your commit hash like 8881a7caa9067471008a8e00750b161f53cdb843>
-```
+### Upload the DevTools binary to CIPD
+- `TARGET_COMMIT_HASH` should be the commit hash you want to release DevTools from
+   - this should match the
+commit hash for the tag you just created
+   - and the [update.sh](https://github.com/dart-lang/sdk/blob/master/third_party/devtools/update.sh)
```

Comment:
Use the update.sh script to build and upload the DevTools binary to CIPD: (then remove where this is stated below).